watchmarketsite/settings.py

```diff
@@ -55,7 +57,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': [ BASE_DIR +'/templates', BASE_DIR +'/users',],
```

Comment:
Calea din `users` pentru ce iti trebuie? Ai setat `APP_DIRS: True` ceea ce inseamna ca va cauta in folderul de `templates` al fiecarei aplicatii.
Partea cu `BASE_DIR + '/templates/` vezi cum se face folosind sistemul de fisiere din `os`, ai exemplul in proiectul de referinta, `django_craiova`.

Reply:
o sa ma uit, asta gasisem pe net ca sa imi mearga template-ul ala de email/register.html, mergea :)

Reply:
'DIRS': [os.path.join(BASE_DIR, "templates"),],
am rezolvat si verificat, merge :)
dau merge?